google/cloud/storage/tests/object_resumable_write_integration_test.cc

```diff
@@ -311,6 +312,15 @@ TEST_F(ObjectResumableWriteIntegrationTest, StreamingWriteFailure) {
       AnyOf(Eq(StatusCode::kFailedPrecondition), Eq(StatusCode::kAborted)))
       << " status=" << os.metadata().status();
 
+  if (UsingEmulator() &&
```

Comment:
It would be more interesting to test the case where this runs against production (vs. the emulator) where the error is parsed and we get something valid, e.g.:

```cc
if (os.metadata().status().code() == StatusCode::kFailedPrecondition) {
  if (UsingEmulator()) {
    EXPECT_THAT(os.metadata().error_info().reason(), Not(IsEmpty()));
    EXPECT_THAT(os.metadata().error_info().domain(), Not(IsEmpty()));
   // maybe something else?
  } else {
    // Your code here
  }
}
```